### ISSUE TITLE
Include statements don't work after metadata statements

### ISSUE THREAD

**Issue by monti-python** [kovetskiy/mark#455]
## What happened?

Adding an `Include` statement right after a `Title` doesn't work, e.g:

```text
<!-- Title: Test -->
<!-- Include: macros.md -->

:children:
```

The only way to solve it to add an additional newline between both tags

## What did you expect to happen?

The include statement above should work

## How can we reproduce the behavior you experienced?

1. Create the following file `macros.md`
```text
<!-- Macro: :box:([^:]+):([^:]*):([^:][\s\S]*?):
    Template: ac:box
    Icon: true
    Name: ${1}
    Title: ${2}
    Body: ${3} -->

<!-- Macro: :children:
    Template: ac:children
-->

<!-- Macro: :updated:
    Template: ac:recently-updated
-->
```

2. Create the following file `test.md`
```
<!-- Title: Test -->
<!-- Include: macros.md -->

:children:
```
3. Export your mark params env vars (`MARK_BASE_URL`, `MARK_SPACE`, `MARK_USERNAME`, `MARK_PASSWORD`)
4. Run `mark -f test.md --include-path "." --dry-run --debug`
5. You should get the below compiled HTML (wrong output):
```html
<p>:children:</p>
```

6. Now add a newline between the title and include statements in the previous `test.md`:
```text
<!-- Title: Test -->

<!-- Include: macros.md -->

:children:
```

7. If you run step `4` again, now you get the right output:
```html
<p><ac:structured-macro ac:name="children"></ac:structured-macro></p>
```

## Further Information (please complete the following information)

* Mark Version (`mark --version`):: 9.12.0
* Mark Parameters: No additional params
* Confluence Hosting: Datacenter
* Confluence Version: N/A
* Environment specific Information: Ubuntu/WSL

**Comment by mrueg** [comment]
Yeah it looks like when it's part of the header, it gets ignored here: https://github.com/kovetskiy/mark/blob/fad9866b416256415a13ad4daf338c8c7347ef2a/pkg/mark/meta.go#L114 probably rather should call ProcessIncludes and feed them back into the file.
